### PR TITLE
English version tweaking

### PR DESCRIPTION
**Pull request by 9214**
_Eek, first PR in foreign repo!_ 😰

English version has been slightly retouched: 
* various typos and spelling corrections;
* `codeblock` formatting where appropiate (mostly `datatype!` names and Red-related terms);
* couple of fixes for broken cross-references and messed-up tables (remained after Markdown to Asciidoc conversion?).

Original writing style and code snippets are untouched.

### CODE REVIEW

File: en/reactivity.adoc [hunk 1]
```diff
@@ -78,7 +78,7 @@ The word `total` above has its value set to the `x + y` expression. Each time th
 
 This variation of Example 3 shows that a global word can also be the target of a reactive relation (though it can't be the source). This form is the closest to that of a spreadsheet's (e.g. Excel) formula model.
 
-NOTE: due to the size of global context, making it reactive (as above with `total`) could have significant performance overhead, though that could be overcome in the future.
+NOTE: due to the size of global context, making it reactive (as above with `total`) could have significant performance overhead, though that could be overcomed in the future.
```
Comment (geekyi) [inline en/reactivity.adoc]: Here too ;)

File: en/preprocessor.adoc [hunk 2]
```diff
@@ -8,7 +8,7 @@
 The Red preprocessor is a dialect of Red, enabling transformation of Red source code using a specific layer on top of regular Red language. Transformations are achieved by inlining preprocessor keywords (called `directives`) inside Red source code. These directives will be processed:
 
 * when the Red source code is compiled
-* when the Red source code is run by `do` function with a file argument
+* when the Red source code is runned by `do` function with a file argument
```
Comment (9214) [inline en/preprocessor.adoc]: I used `runned by` as a synonym for `launched by` here, perhaps it should be... `evaluated with`?

File: en/reactivity.adoc (hunk 1 above)
Comment (9214) [inline en/reactivity.adoc]: @geekyi correct, will fix ASAP.

**Comment by dockimbel** [comment]
Wow, thank you for the many fixes! Unfortunately, I can't merge it anymore as it is in conflict with the current version. Could you please rebase your PR, so I can proceed with the merge?

**Comment by 9214** [comment]
@dockimbel will get to that ASAP, sorry for the delay (boat camping!)

**Pull request merged**